### ISSUE TITLE
Scam websites, maware websites and "human verification" scams not blocked

### ISSUE THREAD

**Issue by randomtroller** [uBlockOrigin/uAssets#8378]
### Prerequisites

- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- [x] I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin

### URL(s) where the issue occurs
1. `0addon.com`
2. `ogtrk.net`
3. `gameglitcher.com`

### Describe the issue
1. It's a website that tries to get the users to download their malicious extension. In the 1st screenshot is the website without uBlock Origin because I had set it to disable JavaScript. In the top left corner next the padlock icon you can see that it tries to load an extension.

2. Seems like an advertising network. Not blocked and I have all filters enabled. In the 2nd and 3rd screenshot we see the website `gameglitcher.com` linking to this website. When I hovered over the "verify now" button, I got sent to an unblocked page that redirects to `ogtrk.net`.

3. A website that sells game "glitches" although there is a free tier that doesn't do anything so most likely the paid ones won't do anything as well. Makes money through unblocked ads and redirections to `ogtrk.net` which redirects to a wider scope of scam sites.
 

### Screenshot(s)
https://i.postimg.cc/L4rS2nry/0addon.png - `0addon.com`
https://i.postimg.cc/WpJRWsh0/ogtrk.png - `ogtrk.net`
https://i.postimg.cc/4xxB01Bj/gameglitcher.png - `gameglitcher.com`

### Versions

- Browser/version: [Firefox 84.0 (64-bit)]
- uBlock Origin version: [1.31.0]

### Settings

- Advanced mode
- I enabled all filters
- JavaScript disabled (I enabled it on all these websites)

### Notes
I was looking around and testing websites (in this example "game hacks") that are known to lead to survey scams and malware. I discovered the 3 domains that were not blocked by any of these lists. I found out that there is still unblocked content (there will always be)

**Comment by mapx-** [comment]
As I said on reddit which are the pages where you get these scam pages ?

Do you get these pages whatever page you visit ? => in this case could be your system is infested by some malware
See the first page connected with some malware => https://malwarefixes.com/threats/remove-0addon-com-pop-up-ads/

**Comment by randomtroller** [comment]
> Do you get the scam stuff whatever page you visit ? => in this case could be your system is infested by some malware

No, I don't get the scam stuff on every page. I found 0addon on these websites when I was looking for unblocked content on purpose. My system is not infected. I listed 0addon.com here because I wanted it to get blocked by one of uBlock origin's filters because it was unblocked.

> As I said on reddit which are the pages where you get (by clicking / redirecting) these scam pages ?
Sorry I didn't understand you. 
When I click on the verification, I get on `fun-clix.mobi` and the website says "enter your phone number to download" while in reality it's a bait for users to sign up for SMS notifcation-thing (it's not quite cheap, really). You are not "downloading" anything. You are signing up for SMS notifications. 

While the service is legit, it's misleading because it looks like it's required to complete the "verification". Screenshot: https://i.postimg.cc/t4wZQhhB/fun-clix.png

The green button says "download now" in Slovenian (because I'm in that region). You are in reality not downloading anything and it's a scam.

**Comment by randomtroller** [comment]
Also, why does it matter where I get redirected to? I'm suggesting that `ogtrk.net` gets blocked. If it gets blocked, the users will not get redirected to the further sites.

For `0addon.com` I think it's obvious because it's malware. I got redirected to it from a youtube to mp3 downloader.

**Comment by mapx-** [comment]
Provide all the steps to be able to see those scam pages:
- original start page
- what to click

> I got redirected to it from a youtube to mp3 downloader

It's what I asked you to provide. Link ?

**Comment by randomtroller** [comment]
> It's what I asked you to provide. Link ?
1. go to ytmp3.cc
2. enter video url
3. click convert
4. download
5. wait to get redirected

You might land on a different page depending on the region.

My starting page is `gameglitcher.com`
For `ogtrk.net`:
1. go to `gameglitcher.com`
2. > list of games >find your game
3. choose any game (in my example Clash Royale)
4. Get redirected to `gameglitcher.com/hacks/(game name hack)`
5. Type any username (can be gibberish)
6. Select any platform
7. Get to the "console"
8. Again, select whatever options on the website (it doesn't matter)
9. Wait for the fake console hack to complete
10. Once it's done, get to the human verification screen
11. Click "verify now"
12. When redirected again, click any of the offers
13. You'll get redirected to a scam page

**Comment by mapx-** [comment]
for ytmp3, add this filter and test again:

`*$script,3p,denyallow=fastly.net|fastlylb.net|jquery.com|hwcdn.net|hcaptcha.com|recaptcha.net|cloudflare.com|cloudflare.net|google.com|googleapis.com|gstatic.com|sharethis.com|stackpathcdn.com|dropbox.com|dropbox-dns.com|ijjiii.is,domain=ytmp3.cc`

**Comment by randomtroller** [comment]
The domain that redirects to those scams gets blocked by EasyList now, It didn't when I tested it.
What about `ogtrk.net` and gameglitcher?

**Comment by mapx-** [comment]
You mean blocking `gameglitcher` ? It makes sense with all requests to download crap

**Comment by mapx-** [comment]
btw for ytmp3, the existent filter in uBO filters list is already blocking any 3-rd party weird scripts
